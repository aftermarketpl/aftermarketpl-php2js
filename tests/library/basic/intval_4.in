$a = false;
return intval($a);
