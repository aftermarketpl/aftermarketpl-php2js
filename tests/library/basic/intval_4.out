var a;
a = '';
return parseInt(a);
